populate/src/main/resources/seed/i18n/de/languageTexts.json

```diff
@@ -136,5 +136,18 @@
   {"keyName": "kitsInPersonCollectedThankYou", "text": "Vielen Dank für Ihre Teilnahme.", "language": "de"},
   {"keyName": "kitTypeSaliva", "text": "Speichelkit", "language": "de"},
   {"keyName": "kitTypeStool", "text": "Stuhlkit", "language": "de"},
-  {"keyName": "kitTypeBlood", "text": "Blutkit", "language": "de"}
+  {"keyName": "kitTypeBlood", "text": "Blutkit", "language": "de"},
```

Comment:
member's choice idea: write a script where you pass in a json blob of new english language texts and it calls google/microsoft translation APIs and returns translated values for all of our supported languages

Reply:
Should we add a comment to section this off as "machine translated" so it's clear next time we need to do any new translations which ones need to be verified?

Reply:
nice idea- i've added `"machineTranslated": true` as a property on these for now